docs/v2-MigrationGuide.md

```diff
@@ -0,0 +1,13 @@
+# Migration guide v2
+
+## Breacking changes
+
+* Merchant's app theme must be decendent of `Theme.MaterialComponents` to operate with "instant" payment components (ex. Paypal, Klarna)
+```xml
+<style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+    <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar"> 
+```
```

Comment:
Maybe we can overcome this breaking change by using the MaterialComponents descendant within our SDK style file.

Reply:
> ❗ Error inflating class `com.google.android.material.button.MaterialButton`

I don't think we can resolve this 🤔